docs/Configuring-Pantheon/Using-Configuration-File.md

```diff
@@ -40,10 +40,13 @@ Specific differences between the command line and the TOML file format are:
     
     rpc-http-host="5.6.7.8"
     rpc-http-port=5678
+    rpc-http-apis=["ETH","NET","WEB3"]
+    rpc-http-cors-origins=["*"]
+    host-whitelist=["127.0.0.1","localhost"]
```

Comment:
This 127.0.0.1 and localhost should be the default for host-whitelist so shouldn't be required here.

rpc-http-apis should default to ETH, NET and WEB3 as well.

So you should only need the rpc-http-cors-origins and then only if you want to open your local node to every website you visit.

Reply:
true, the `rpc-http-apis` I was using was actually a long list of non-default apis.